app/src/main/res/values/strings.xml

```diff
@@ -63,111 +63,111 @@
     <string name="txt_level10_name">Champion</string>
     <string name="level_up_message">Congrats! You have just achieved level {{*{{n_level}}*}}.</string>
     <!-- Settings -->
-    <string name="btn_open_tutorial_settings">Open Tutorial again</string>
+    <string name="btn_open_tutorial_settings">Open tutorial again</string>
     <string name="txt_project_github_settings">Project open-source on GitHub</string>
     <string name="txt_contributors">main contributors</string>
     <string name="txt_developed_by">developed by</string>
     <string name="toast_language_changed">Language changed correctly to {{*{{lang}}*}}</string>
     <string name="settingsLanguage">Language</string>
-    <string name="settingsListen">Listen section</string>
-    <string name="settingsSpeak">Speak section</string>
+    <string name="settingsListen">Listen</string>
```

Comment:
Here is the section in the Settings, so I think is better specify "section" ...?

Reply:
This doesn't follow the design of other apps.

For example, in Riot, the headers of each section are called "user settings", "contacts", "user interface", "home display", "flair" and so on, not "user settings section", "contacts section", "flair section".

In F-Droid settings, the headers are "my apps", "updates", "display" and so on, not "my apps section", "updates section", "display section".

It's obvious that they are sections — writing out "sections" agains is not only redundant, it's inconsistent with other apps.

Reply:
Oh, okok. It's true 😄